BraintreeCore/src/main/java/com/braintreepayments/api/UnstableApi.kt

```diff
@@ -0,0 +1,4 @@
+package com.braintreepayments.api
+
+@RequiresOptIn
+annotation class UnstableApi
```

Comment:
Should we maybe say "ExperimentalApi" or "BetaFeature"? I think we should establish a convention with this annotation that we use throughout others

Reply:
I wanted to put up a PR to get ideas on what we might want to call this.
I'm open to suggestions.

Reply:
How about just `@Beta` or `@Unstable`? `@Experimental`sounds like A/B testing.

Reply:
Can we do a 2nd pass about this naming in stand? Would be great to hammer down what we want to call this since we will need to use it across several upcoming features.

Reply:
Would `BetaExperiementalApi` or `ExperiementalBetaApi` be too verbose?

Reply:
I like `ExperiementalBetaApi`. I think most people will use autocomplete in their IDE to add the annotation.